core/src/main/scala/cats/data/EitherT.scala

```diff
@@ -77,6 +77,13 @@ final case class EitherT[F[_], A, B](value: F[Either[A, B]]) {
                                                          applicativeG: Applicative[G]): G[EitherT[F, C, D]] =
     applicativeG.map(traverseF.traverse(value)(axb => Bitraverse[Either].bitraverse(axb)(f, g)))(EitherT.apply)
 
+  def biFlatMap[AA >: A, BB >: B](fa: A => EitherT[F, AA, BB],
+                                  fb: B => EitherT[F, AA, BB])(implicit F: FlatMap[F]): EitherT[F, AA, BB] =
+    EitherT(F.flatMap(value) {
+      case Left(a)  => fa(a).value
+      case Right(a) => fb(a).value
```

Comment:
would not it be more intuitive to say `case Right(b) => fb(b).value` at L.84 to match with the type `B` fb expects?